### PR TITLE
Finishes vampire for role datums

### PR DESCRIPTION
**Pull request by ShiftyRail**
- Adds spells icons (courtesy of Blithering and Paradise)
- Fixes the issue with XRay blinking
- Adds the ability to spawn a cape

With that, vampire is complete.
Until bugs are found anyway

vampsells

### CODE REVIEW

File: code/modules/mob/living/carbon/human/life/handle_regular_hud_updates.dm [hunk 1]
```diff
@@ -95,16 +95,28 @@
...
-		change_sight(removing = SEE_TURFS|SEE_MOBS|SEE_OBJS)
+		// Moiving this "see invisble" thing here so that it can be overriden by xrays, vampires...
 
-		var/datum/organ/internal/eyes/E = src.internal_organs_by_name["eyes"]
-		if(E)
-			see_in_dark = E.see_in_dark //species.darksight
-		else
-			see_in_dark = species.darksight
+		// Vampire bandaid. I'm sorry.
+		var/datum/role/vampire/V = isvampire(src)
```
Comment (MadmanMartian) [inline code/modules/mob/living/carbon/human/life/handle_regular_hud_updates.dm]: This plus the block below = Absolutely heretical.

Comment (ShiftyRail) [reply code/modules/mob/living/carbon/human/life/handle_regular_hud_updates.dm]: I could not find a way to do it that's not retarded.
The role process() is called before the life().

Comment (MadmanMartian) [reply code/modules/mob/living/carbon/human/life/handle_regular_hud_updates.dm]: Honestly this entire portion of lifecode needs a rewrite, why the hell is sight change being called every life tick? Makes no sense.

**Comment by Irradiation** [comment]
It's not too late to scrap vampire from datums, Shifty. Don't fall for the sunken cost meme.

**Comment by Irradiation** [comment]
GOD DAMNIT, YOU WERE MEANT TO BE THE ONE!

**Pull request merged**